src/pharmpy/plugins/nonmem/__init__.py

```diff
@@ -38,7 +39,10 @@ class NONMEMConfiguration(config.Configuration):
         list,
     )
     default_nonmem_path = config.ConfigItem(
-        pathlib.Path(''), 'Full path to the default NONMEM installation directory', cls=pathlib.Path
+        pathlib.Path(''),
+        'Full path to the default NONMEM installation directory',
+        cls=pathlib.Path,
+        transform=ux.normalize_user_given_path,
```

Comment:
Not sure this belongs here.

Reply:
I think you don't need the extra transform argument here. `cls` could be set to any callable returning an object of the class.